### PR TITLE
New post

### PR DESCRIPTION
**Pull request by junaidshaikh-js**
This PR includes 

- Users can add a new post 
- User can view all posts on the home page in the order posts are created (which will be changed to posts by the user following )
- Users can view posts posted by them in the post tab on the profile page
- Navigation and sidebar are now sticky
- some code refactoring

### CODE REVIEW

**Review by abhij1607 — APPROVED** [review]
Neat & Clean code
Logic well implemented

Can be merged

File: src/app/App.js [hunk 1]
```diff
@@ -13,6 +19,21 @@ import {
...
+  }, [dispatch, token]);
+
+  useEffect(() => {
+    if (token) dispatch(getAllPosts());
+  }, [dispatch, token]);
+
+  useEffect(() => {
+    if (token) dispatch(getAllUsers());
+  }, [dispatch, token]);
+
```
Comment (sanket4120) [inline src/app/App.js:36]: you can create custom hook for this

**Review by sanket4120 — APPROVED** [review]
Rest looks good to me

File: src/app/App.js (hunk 1 above)
Comment (junaidshaikh-js) [reply src/app/App.js:36]: Thank you, I will refractor it later on.

**Pull request merged**